string input_msg
---
panda_mover/ListOfPoses[] trajectoryList
